src/Symfony/Component/Intl/DateFormatter/IntlDateFormatter.php

```diff
@@ -635,7 +635,7 @@ protected function getDefaultPattern()
         if (self::NONE !== $this->timetype) {
             $patternParts[] = $this->defaultTimeFormats[$this->timetype];
         }
-        $pattern = implode(' ', $patternParts);
+        $pattern = implode(', ', $patternParts);
```

Comment:
Well, the stub implementations mimicks ICU 5.1 currently. Are you sure this is what ICU 5.1 is doing ?

Tests relying on a particular ICU behaviors should use the test helper to be skipped when a different ICU version is used, and should be written against the ICU version used for the stub implementation.
I know that we have mistakes in the Form testsuite for recent tests (e.g. features relying on Intl but not implemented by @webmozart), but the testsuite should be changed to expect the same ICU version than other tests instead of changing the stub implementation (which would not match any ICU version anymore)